<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnConectar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Consolas&quot;;
background: rgb(0,128,0);
color: rgb(85,255,0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Consolas&quot;;
background: rgb(0,128,0);
color: rgb(85,255,0);
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>36</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>87</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>139</y>
      <width>109</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>158</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Base de Datos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUser">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>89</y>
      <width>163</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid #33FF38;
background-color: rgb(0,0,0);
color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>root</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>129</y>
      <width>163</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid #33FF38;
background-color: rgb(0,0,0);
color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>root</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDB">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>179</y>
      <width>163</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid #33FF38;
background-color: rgb(0,0,0);
color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>empresa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHost">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>163</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid #33FF38;
background-color: rgb(0,0,0);
color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>txtUser</zorder>
   <zorder>txtPassword</zorder>
   <zorder>txtDB</zorder>
   <zorder>txtHost</zorder>
   <zorder>btnConectar</zorder>
   <zorder>btnCancelar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
